src/main.rs

```diff
@@ -4,15 +4,37 @@ use anyhow::{self, Context};
 use clap::Parser;
 use tokio::{select, signal};
 use tracing_subscriber::prelude::*;
-use wasmcloud_host::WasmbusHostConfig;
+
+mod config;
+
+#[allow(clippy::wildcard_imports)]
+use config::*;
+use wasmcloud_host::wasmbus::HostConfig;
```

Comment:
Could you move this import back together with all the other imports?

Reply:
@connorsmith256 I understand that there is a particular order in other repositories, in ordering in this repository is the following:

- pub use super...
- pub use crate...
- pub use core...
- pub use std...
- pub use everything-else (including other crates in the same workspace)

- use super...
- use crate...
- use core...
- use std...
- use everything-else (including other crates in the same workspace)

Let's please follow the format already used throughout the codebase in this repository and we can always refactor it later. There is no reason to introduce inconsistencies here.

As a part of the refactoring we should introduce a development guideline to avoid such situations in the future